mindmeld/text_preparation/stemmers.py

```diff
@@ -72,6 +65,10 @@ def stem_word(self, word):
 
 
 class SnowballNLTKStemmer(Stemmer):
+
+    def __init__(self, language=None):
+        self.language = language
```

Comment:
why do you need language here?

Reply:
The SnowballNLTK Stemmer can support multiple languages so it must be specified during instantiation.